System1/System1_NavigateTo_WI.xaml

```diff
@@ -0,0 +1,107 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="System1_NavigateTo_WI" mva:VisualBasic.Settings="{x:Null}" sap:VirtualizedContainerService.HintSize="1252,770" sap2010:WorkflowViewState.IdRef="System1_NavigateTo_WI_1" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
```

Comment:
Redundant workflow, you don't need a separate workflow just for a single click. you might move click items activity directly in ExtractWorkItems workflow